applications/serial_lte_modem/doc/FOTA_AT_commands.rst

```diff
@@ -47,10 +49,17 @@ Syntax
 
 .. note::
 
-   When doing modem FOTA, erasing the modem DFU area is optional since the update process will automatically erase the area if needed.
+   When doing modem delta FOTA, erasing the modem DFU area is optional since the update process will automatically erase the area if needed.
 
    However, this leads to the command starting the update taking longer to complete, and also leaves the connection to the FOTA server idle while the area is being erased, which can provoke issues.
 
+.. note::
+
+   When doing full modem FOTA, external flash is used to store the firmware.
+   The external flash is ereased automatically after a new firmware activation.
+
+   The activation of a new full modem firmware is done identically to a modem delta update, by resetting either the whole device or only the modem.
```

Comment:
'Activation' sounds weird to me here, but I can't find a clear precedent on the Cloud side.

My gut says *application* is a better word based on https://docs.nrfcloud.com/Devices/FirmwareUpdate/FOTAOverview.html, but I've also realized we don't ever actually say what this step is called. The entire process is deployment, and in some APIs the act of installing new firmware is called *application*, but that's still more general than what _this_ doc is referring to.

Reply:
Switching 'activation' and 'application' would be confusing. Here 'activation' means the action to be made and 'application' the software itself which is downloaded.